.env.dev

```diff
@@ -15,7 +15,8 @@ VITE_CLIENT_PORT = 8000 # Client port
 VITE_NODE_URL = http://localhost:3013 # Node URL
 VITE_COMPILER_URL = http://localhost:3080
 VITE_WS_URL = ws://localhost:3014/channel
-VITE_NETWORK_ID = ae_devnet
 VITE_FAUCET_PUBLIC_ADDRESS = ak_2iBPH7HUz3cSDVEUWiHg76MZJ6tZooVNBmmxcgVK6VV8KAE688
+VITE_FAUCET_LOCAL_ADDRESS = ak_2dATVcZ9KJU5a8hdsVtTv21pYiGWiPbmVcU1Pz72FFqpk9pSRR # This refers to local node genesis account
```

Comment:
Therefore, I think there is no need to check whether a local node is running or not, since the variable name will be the same whether if you use this .env or the uat one